client/blocks/inline-help/popover.jsx

```diff
@@ -381,6 +382,10 @@ function mapStateToProps( state, { moment } ) {
 		].includes( getActiveTheme( state, siteId ) ) &&
 		moment( getSiteOption( state, siteId, 'created_at' ) ).isAfter( '20190314' );
 
+	const isAllowedToUseClassic =
+		! isSiteUsingFullSiteEditing( state, siteId ) && ! isUsingGutenbergPageTemplates;
+	const showOptOut = optInEnabled && isGutenbergEditor && isAllowedToUseClassic;
+
```

Comment:
I like this approach  - easier to reason about this way rather than combining 'trues' with '! trues' in a single line

Reply:
Thanks! I think so too :)